priv/vars.sql

```diff
@@ -2,4 +2,4 @@
 SELECT v.name, v.type, v.value from vars_inventory v order by name
 
 -- :vars_get
-SELECT v.name, v.type, v.value from vars_inventory v where v.name = ?
+SELECT v.name, v.type, v.value from vars_inventory v where v.name = ?;
```

Comment:
No.. I meant replace ? with $1?

Reply:
I guess I wasn't sure where in esql file I should put a semicolon if anywhere? That's what I was trying to figure out